security/s2a/internal/crypter/ciphersuite.go

```diff
@@ -0,0 +1,102 @@
+package crypter
+
+import (
+	"crypto/sha256"
+	"golang.org/x/crypto/sha3"
+	s2a_proto "google.golang.org/grpc/security/s2a/internal"
```

Comment:
Is this importing everything in `internal`? I think you only need `common.pb.go`.

Reply:
I think importing `internal` is as granular as it gets since we have to import entire packages in Go. However, I noticed that you merged a PR that moved the `common.pb.go` file over. Once I rebase, I assume I won't have to import `internal` anymore.

Reply:
SGTM!